tests/settings/test_settings.py

```diff
@@ -18,3 +18,28 @@ def test_invalid_storage() -> None:
         assert False, f"'validate_settings' raised an exception {exc}"
     finally:
         cluster[0]["storage_sets"].remove("non_existing_storage")
+
+
+def test_topics_sync() -> None:
+    from copy import deepcopy
+
+    from snuba.settings.validation import validate_settings
+    from snuba.utils.streams.topics import Topic
```

Comment:
Is there a reason these imports are inside the test itself?

Reply:
I was following the style of the above test. I think the idea is to place relevant imports where they belong, so the imports that are at the top of the file are relevant to _every_ test.

Reply:
You should not need that. 
The settings module cannot import anything else as it must be possible to import it from anywhere. 
Though this should not be a problem in the test. 

I would try to move all the import at the beginning of the file. This is for both tests.